2022-01-31 16:19:16,251 INFO    Thread-16 :222084 [wandb_setup.py:_flush():71] setting env: {'project': 'pytorch-sweeps-demo', 'entity': 'paulina', 'root_dir': '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10', 'run_id': '975u9i7u', 'sweep_param_path': '/blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/sweep-o8viijca/config-975u9i7u.yaml', 'sweep_id': 'o8viijca'}
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [wandb_init.py:_log_setup():371] Logging user logs to /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_161916-975u9i7u/logs/debug.log
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [wandb_init.py:_log_setup():372] Logging internal logs to /blue/fsaeed/paulinaacostacev/ImageClassifierCIFAR10/wandb/run-20220131_161916-975u9i7u/logs/debug-internal.log
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [wandb_init.py:init():404] calling init triggers
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [wandb_init.py:init():409] wandb.init called with sweep_config: {'batch_size': 195, 'epochs': 1, 'layer1_size': 256, 'layer2_size': 512, 'learning_rate': 0.02523039639596334, 'optimizer': 'adam'}
config: {}
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [wandb_init.py:init():460] starting backend
2022-01-31 16:19:16,252 INFO    Thread-16 :222084 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-31 16:19:16,254 INFO    Thread-16 :222084 [backend.py:ensure_launched():216] starting backend process...
2022-01-31 16:19:16,316 INFO    Thread-16 :222084 [backend.py:ensure_launched():221] started backend process with pid: 227851
2022-01-31 16:19:16,318 INFO    Thread-16 :222084 [wandb_init.py:init():469] backend started and connected
2022-01-31 16:19:16,319 INFO    Thread-16 :222084 [wandb_run.py:_config_callback():956] config_cb None None {'batch_size': 195, 'epochs': 1, 'layer1_size': 256, 'layer2_size': 512, 'learning_rate': 0.02523039639596334, 'optimizer': 'adam'}
2022-01-31 16:19:16,321 INFO    Thread-16 :222084 [wandb_init.py:init():533] updated telemetry
2022-01-31 16:19:16,322 INFO    Thread-16 :222084 [wandb_init.py:init():563] communicating current version
2022-01-31 16:19:18,076 INFO    Thread-16 :222084 [wandb_init.py:init():568] got version response 
2022-01-31 16:19:18,076 INFO    Thread-16 :222084 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-31 16:19:18,237 INFO    Thread-16 :222084 [wandb_init.py:init():606] starting run threads in backend
2022-01-31 16:19:19,850 ERROR   Thread-16 :222084 [wandb_init.py:init():958] error
Traceback (most recent call last):
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 931, in init
    run = wi.init()
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 611, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 423, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/home/paulinaacostacev/miniconda3/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
Exception
